lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -259,6 +259,19 @@ static bool recursiveType(const StructType *ST, const Type *Ty) {
   return Run(Ty);
 }
 
+static bool isManifestConstant(const Constant *c) {
+  if (isa<ConstantData>(c)) {
+    return true;
+  } else if (isa<ConstantAggregate>(c) || isa<ConstantExpr>(c)) {
+    for (const Value *subc : c->operand_values()) {
```

Comment:
The same for "subc" here, should change to "Subc". The error log from pre-ci system:
Found tidying errors!
/home/travis/build/KhronosGroup/SPIRV-LLVM-Translator/lib/SPIRV/SPIRVWriter.cpp:262:48: warning: invalid case style for parameter 'c' [readability-identifier-naming]
static bool isManifestConstant(const Constant *c) {
                                               ^~
                                               C
/home/travis/build/KhronosGroup/SPIRV-LLVM-Translator/lib/SPIRV/SPIRVWriter.cpp:266:23: warning: invalid case style for variable 'subc' [readability-identifier-naming]
    for (const Value *subc : c->operand_values()) {
                      ^~~~
Thanks very much.